freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/kerberosmgmt/model/CleanupRequest.java

```diff
@@ -0,0 +1,37 @@
+package com.sequenceiq.freeipa.api.v1.kerberosmgmt.model;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.sequenceiq.service.api.doc.ModelDescriptions;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel("CleanupV1Request")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CleanupRequest {
```

Comment:
we already have a cleanuprequest. Could you rename this one?

Reply:
ok